helix-term/src/commands/typed.rs

```diff
@@ -334,6 +334,10 @@ fn write_impl(
     let (view, doc) = current!(cx.editor);
     let path = path.map(AsRef::as_ref);
 
+    if doc.detect_readonly() && !force {
+        bail!("'readonly' option is set (add ! to overwrite)")
```

Comment:
```suggestion
        bail!("'file is readonly' option use ! to overwrite")
```

Reply:
I'll change it to `file is readonly, use ! to overwrite`, I assume that's what you meant